vipps-recurring-api.md

```diff
@@ -204,7 +204,7 @@ For a `"transactionType": "RESERVE_CAPTURE"` setup, the normal flow would be:
 | [Create a charge](#create-a-charge)             | Create a new charge for an agreement.                 | [`POST:/recurring/v2/agreements/{agreementId}/charges`][create-charge-endpoint]                     |
 | [Retrieve a charge](#retrieve-a-charge)         | Retrieve all details of a charge.                     | [`GET:/recurring/v2/agreements/{agreementId}/charges/{chargeId}`][fetch-charge-endpoint]            |
 | Capture a charge                                | Each charge must first be created, then captured.     | [`POST:/recurring/v2/agreements/{agreementId}/charges/{chargeId}/capture`][capture-charge-endpoint] |
```

Comment:
We are missing a paragraph for capture charge and refund charge. Making a JIRA task for it. Can be made part of the v3 updates.